_config.yml

```diff
@@ -316,3 +316,271 @@ t:
     contact:
       title: "Follow the Pyronear adventure"
       content: "To stay updated about our progress or to join the adventure, you can also reach us on social media!"
+  ca:
+    label: ca
+    icon: 🇨🇦
```

Comment:
This is the canadian flag, not the catalan/valencian flag

Reply:
There is an error here, I had not seen what the country's flag represents. Where can I find the flag of Catalonia?

Reply:
If it can be found, I imagine it'll be among the unicode emoji : https://unicode.org/emoji/charts/full-emoji-list.html

Reply:
I have been checking the emojis and the Catalan flag is not regulated by the Unicode Consortium, is there an alternative?

Reply:
@frgfm @MateoLostanlen Do you know any solution to be able to add the Catalan flag? It's the only thing left to fix. thanks!

Reply:
@psd24 it seems available [here](https://iconduck.com/emojis/47135/catalonia-flag)